src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/Program.cs

```diff
@@ -31,13 +32,20 @@ private void ProcessRunOptions(string[] args)
             //  -   # factories to pool
             //  -   # channels to pool
             //  -   etc
-            TestHelpers.SetHostAndProtocol(useHttp: false, hostName: "localhost", appName: "WcfService1");
+            var hostName = Environment.GetEnvironmentVariable("hostName");
```

Comment:
I thought about using it, but I'd like the stress & perf test code to not take *any* dependencies on the test source tree. 
One reason is that I often find myself compiling and running these tests on the desktop WCF (mostly for perf but sometimes for stress too). It makes it easier to give standalone repros to other teams too.